lib/errors.js

```diff
@@ -73,6 +73,8 @@ class HTTPError extends Error {
 
     res.write(content);
     res.end();
+
+    Log.debug(err);
```

Comment:
Is `err` always valid in this code path? If it comes in as `null` or `undefined` does the logic in lines 64 through 66 above still work?